tests/test_ply.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2018 Peter Petrik (zilolv at gmail dot com)
```

Comment:
Probably you want to own the copyright yourself :)

Reply:
Done in the next commit